### ISSUE TITLE
100% confidence for wrong utterance + api 500 errors for multiple wit apps

### ISSUE THREAD

**Issue by CannonFodderr** [wit-ai/wit#2363]
Hello, we encountered few issues in multiple apps lately.

1. Some apps always return confidence 100% for every utterance, 
**i.e.** 
I create intent about **money** with few utterances about **money** and then submit a message **ice cream**, 
the **ice cream** utterance will return confidence 100% and the ice cream utterance will now attached to the money intent.

2. We tried to get the ice cream utterance out of scope, this resulted in the following error:
status code: 500
```
{
    "error": "Something went wrong. We've been notified",
    "code": "unknown"
}
```

2. We tried to fix this by exporting and importing the data to new app, this resulted in both apps returning the error.

3. This error sometimes happen randomly for multiple apps.
Example app id: 1184234872397015 (But we have quite a few apps with this behavior )

4. After deleting the newly imported app still the old app returns the error.
5. Long training times, sometimes it takes hours to app show it is fully trained.

Please advise.

**Comment by doggeral** [comment]
> Some apps always return confidence 100% for every utterance

You can try to add some out of domain samples to enable the model capture your intent pattern. This issue is more likely when you have small training samples.

> We tried to fix this by exporting and importing the data to new app, this resulted in both apps returning the error.

We are upgrading the training infra recently, you can try to create your app from the scratch instead of importing the previously existing apps. 

> This error sometimes happen randomly for multiple apps.

Seems like the training samples issue. I update the training sample and retrain, it should work.

> Long training times, sometimes it takes hours to app show it is fully trained.

Could you provide the app id?

**Comment by CannonFodderr** [comment]
@doggeral Thanks for the reply
The previous app I send now seems to work ok but we have lots of other apps with same or similar issue:

1. App id: 651606172847922 
Hello world is out of scope
I send request message "Hello world" get response:

```
{
    "error": "Something went wrong. We've been notified",
    "code": "unknown"
}
```

2. App Id: 987727735495683, 
This app have 5 out of scope utterances in English and 3 utterances in Japanese attached to an intent.
"Hello world" utterance  is marked as out of scope

When I do a message request with "Hello world" I get the intent with Japanese in it as 98% confidence.

I then exported the data from the app and created a new app: 478052880433022 , I manually created the utterances again from the previous app data, after app is manually recreated then issue seems to be resolved.
Is there any way we can resolve it without manually creating multiple apps ? it is also an issue because exporting some languages data to json (Like Japanese) will result in utterance no showing correctly in json and needs to be converted back to the language. (See attached)

![Capture](https://user-images.githubusercontent.com/36086310/154476718-b14c2d32-484e-4432-81d9-b0e1367da190.PNG)
![Capture2](https://user-images.githubusercontent.com/36086310/154476723-99ab1ca0-ae63-48a2-b6e6-fe3b24f10d77.PNG)
![Capture3](https://user-images.githubusercontent.com/36086310/154481302-4a32cec5-25ed-49a7-831a-0d0bc94b459c.PNG)

**Comment by doggeral** [comment]
I see, please provide the list of app ids, we can do the batch operation for you.

**Comment by CannonFodderr** [comment]
@doggeral Thanks here is the app ids we know of, but it is only the ones reported to us by the users complains. Is there away you can spot what is the issue with the apps which causes them to malfunction ?

**APP IDS:**
1184234872397015
978810119619693
233334358781880
271386008408980
651606172847922
987727735495683
286268900241585
1292133577973221

**Comment by doggeral** [comment]
@CannonFodderr Thanks for providing the list. All you apps have been running on the new model architecture now.

**Comment by leslieam** [comment]
Thanks for the quick response.
We'll check it out.

**Comment by CannonFodderr** [comment]
@doggeral after checking the apps only one of them is now fixed: 1184234872397015

The rest of the apps are still malfunctioning: 

978810119619693
233334358781880
271386008408980
651606172847922
987727735495683
286268900241585
1292133577973221

plus customer created 2 new apps which also gets these issues:

353659586134377
321292556446914
